integration_tests/dbt_project.yml

```diff
@@ -14,11 +14,13 @@ vars:
     twilio_usage_record_identifier:   "twilio_usage_record_data"
     twilio_call_identifier:   "twilio_call_data"
     twilio_message_identifier:   "twilio_message_data"
-
 dispatch:
   - macro_namespace: dbt_utils
     search_order: ['spark_utils', 'dbt_utils']
 
+models:
```

Comment:
More of a question: There weren't any validation tests added, so was this just for future use? Did we think it wouldn't have been useful to add a row comparison test given the changes weren't significant enough to the code?

Reply:
Yeah! Yup just for future use. No validation tests for this PR since this isn't really changing any model.